src/themes/eui-amsterdam/overrides/_filter_group.scss

```
@@ -0,0 +1,3 @@
+.euiFilterButton.euiButtonEmpty {
```

Because the Amsterdam `filter_group` file comes after `button_empty` the singular class `.euiFilterButton` will always come after `.euiButtonEmpty` utilizing the cascade effect of CSS and making this chaining unecessary.

```suggestion
.euiFilterButton {
```